src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
Well, its a debug message, so its not important enough to argue about any more :)

I totally agree with the point of the whole PR, I'm just trying to help you keep its integrity, not allow cracks where ctags appears at a user level.  The fact that the parsers are ctags based is just a current implementation artifact.  For example @codebrainz is working on identifying symbols in c/c++ code using libclang, thats not ctags related.

In the new terminology "symbols parser" is the parser that generates the symbol list, just like the existing terminology of a "tags parser" generates a tags list, its doesn't "parse tags".

We could accept either key in the config file "tag_parser" or "symbol_parser" so we remain backward compatible.